pymchelper/utils/__init__.py

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
Yes, we need strong copyright on it. Out emptiness may not be copied nor reproduced without our authorization..